steps/cli/oc.go

```diff
@@ -138,6 +138,10 @@ var _ = gauge.Step("Delete <resourceType> named <name>", func(resourceType, name
 	oc.DeleteResource(resourceType, name)
 })
 
+var _ = gauge.Step("Uninstall TektonHub", func() {
```

Comment:
IMHO we don't need a new step, you can directly extend step `Uninstall Operator`, i.e. function  `pkg/operator/operator.go#Uninstall`